springfield/firefox/templates/firefox/enterprise/index.html

```diff
@@ -271,7 +267,7 @@ <h4 class="enterprise-download-subtitle">{{ ftl('firefox-enterprise-support') }}
       </div>
 
       <p class="enterprise-languages">
-        {{ ftl('firefox-enterprise-download-firefox-esr-or-rapid', firefox_all=firefox_url('desktop', 'all', 'esr')) }}
+        {{ ftl('firefox-enterprise-download-firefox-or-esr', firefox_all=firefox_url('desktop', 'all', 'esr')) }}
```

Comment:
set the href attrs here better?

Reply:
done in 568f0d8374898c474007d85bd3c4648b5b81c89f